<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialog</class>
 <widget class="QDialog" name="InfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>410</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(36, 36, 36);
background-image: url(:/images/sidepanelimage_info.jpg);
background-repeat: no-repeat;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="container" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget #container {
background-color: rgb(38, 38, 38);
border-radius: 5px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="visitWebsite">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
background-color: rgb(67, 67, 67);
border-style: inset;
border-radius: 3px;
text-decoration: none;
}
QPushButton:hover
{
background-color: rgb(104, 104, 104);
border-style: inset;
}
</string>
              </property>
              <property name="text">
               <string>Visit Website</string>
              </property>
              <property name="icon">
               <iconset resource="../../app/icons.qrc">
                <normaloff>:/icons/world.svg</normaloff>:/icons/world.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="viewTutorials">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
background-color: rgb(67, 67, 67);
border-style: inset;
border-radius: 3px;
text-decoration: none;
}
QPushButton:hover
{
background-color: rgb(104, 104, 104);
border-style: inset;
}
</string>
              </property>
              <property name="text">
               <string>View Tutorials</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/educational-video.svg</normaloff>:/icons/educational-video.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget #widget_7 {
background-color: rgb(58, 58, 58);
border-bottom: 1px solid rgb(0, 170, 255);
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>5</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>2000</width>
                   <height>2000</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Recent Projects</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="recentProjects">
              <property name="styleSheet">
               <string notr="true">/*list view style*/

QListView {
	color: white;
	background-color: rgb(39, 39, 39);
}


QListView::item {
	background-color: rgb(139, 139, 139);
    border-bottom: 1px solid rgb(46, 46, 46);
	padding: 5px 15px 5px 15px;
	border-top-left-radius: 2px;
	border-bottom-left-radius: 2px;
}

QListView::item:hover {
	background-color: rgb(181, 181, 181);
	padding: 10px 10px 10px 10px;
	text-decoration: none;
}

QListView::item:selected {
	background: #1f90ff;
	padding: 10px 10px 10px 10px;
	text-decoration: none;
}

/*--------------------  list view scrollbar style  ------------------------------*/

 QScrollBar:vertical {
	 background-color: rgb(180, 180, 180);
     width: 15px;
     margin: 22px 0 22px 0;
 }

 QScrollBar::handle:vertical {
	background-color: rgb(97, 97, 97);
     min-height: 20px;
	 border-radius: 5px;
	 margin-left: 2px;
	 margin-right: 2px;
 }

 QScrollBar::handle:vertical:hover {
	background-color: rgb(110, 110, 110);
     min-height: 20px;
	 border-radius: 5px;
	 margin-left: 2px;
	 margin-right: 2px;
 }

 QScrollBar::add-line:vertical {
	 background-color: rgb(180, 180, 180);
     height: 30px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical {
	 background-color: rgb(180, 180, 180);
     height: 30px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }

QScrollBar::up-arrow:vertical
{
    image: url(:/icons/up-arrow.svg);
}
QScrollBar::up-arrow:vertical:hover
{
	background-color: rgb(162, 162, 162);
    image: url(:/icons/up-arrow.svg);
}

QScrollBar::down-arrow:vertical
{
    image: url(:/icons/down-arrow.svg);
}

QScrollBar::down-arrow:vertical:hover
{
	background-color: rgb(162, 162, 162);
    image: url(:/icons/down-arrow.svg);
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string> Always show on startup</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
background-color: rgba(155, 155, 155,0);
border-radius:10px;
border: 2px solid  rgb(223, 223, 223);
color: rgb(255, 255, 255);
}
QPushButton:hover
{
background-color: rgb(223, 223, 223);
border: 2px solid  rgb(223, 223, 223);
color: rgb(115, 115, 115);
}
</string>
           </property>
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../app/icons.qrc"/>
 </resources>
 <connections/>
</ui>
